tests/test_entities.py

```diff
@@ -398,6 +400,19 @@ def test_no_such_path_error(self):
                 with self.assertRaises(NoSuchPathError):
                     entity(self.cfg_610).path(which=which)
 
+    def test_os_default_template(self):
+        """ Test :meth:`nailgun.entities.OSDefaultTemplate.path`
```

Comment:
My bad, according to travis log sphinx can't build a link as can't find inherited method. Pls revert back to 
```
``nailgun.entities.OSDefaultTemplate.path``
```

Reply:
Yea, just figured that out, too. Will revert. Thanks!